NMSReportingSuite/src/main/java/com/beehyv/nmsreporting/controller/UserController.java

```diff
@@ -643,26 +643,16 @@ private int countLocationsWithHighCompletion(List<AggregateCumulativeMA> startDa
         int count = 0;
         Map<Long, AggregateCumulativeMA> startMap = new HashMap<>();
 
-        LOGGER.debug("we are inside the locationwithcompletioncode");
         for (AggregateCumulativeMA start : startData) {
             startMap.put(start.getLocationId(), start);
         }
 
-        Long registered = 0L;
-        Long completed =0L;
-
         for (AggregateCumulativeMA end : endData) {
-
             AggregateCumulativeMA start = startMap.get(end.getLocationId());
             if (start != null) {
-                try{
-                    registered = (long) (end.getAshasRegistered() - start.getAshasRegistered());
```

Comment:
comments

Reply:
I will add comments and refactoring in another pr.